package.json

```diff
@@ -17,51 +17,49 @@
   },
   "private": true,
   "dependencies": {
-    "@angular/animations": "^9.0.4",
-    "@angular/cdk": "^9.1.0",
-    "@angular/common": "^9.0.4",
-    "@angular/core": "^9.0.4",
-    "@angular/forms": "^9.0.4",
-    "@angular/http": "^6.1.7",
-    "@angular/localize": "^9.0.4",
-    "@angular/material": "^9.1.0",
-    "@angular/platform-browser": "^9.0.4",
-    "@angular/platform-browser-dynamic": "^9.0.4",
-    "@angular/router": "^6.1.7",
+    "@angular/animations": "^11.2.14",
+    "@angular/cdk": "^11.2.13",
+    "@angular/common": "^11.2.14",
+    "@angular/core": "^11.2.14",
+    "@angular/forms": "^11.2.14",
+    "@angular/localize": "^11.2.14",
+    "@angular/material": "^11.2.13",
+    "@angular/platform-browser": "^11.2.14",
+    "@angular/platform-browser-dynamic": "^11.2.14",
+    "@angular/router": "^11.2.14",
     "@auth0/angular-jwt": "^2.0.0",
     "@ckeditor/ckeditor5-angular": "^1.1.0",
-    "@ng-bootstrap/ng-bootstrap": "^6.0.0",
-    "@tinymce/tinymce-angular": "^3.0.1",
+    "@ng-bootstrap/ng-bootstrap": "^9.1.3",
+    "@tinymce/tinymce-angular": "^4.2.4",
     "@types/geojson": "^7946.0.2",
     "@types/leaflet": "^1.2.6",
     "bootstrap": "^4.1.3",
-    "codelyzer": "^4.4.4",
+    "codelyzer": "^6.0.2",
     "core-js": "^2.5.7",
-    "hammerjs": "^2.0.8",
     "keycloak-js": "^19.0.1",
     "leaflet": "^1.3.1",
     "lodash": "^4.17.19",
     "material-components-web": "^5.1.0",
     "moment": "^2.29.2",
-    "moment-timezone": "0.5.28",
+    "moment-timezone": "0.5.35",
     "mygovbc-bootstrap-theme": "^0.4.1",
     "ngx-cookie-service": "^2.4.0",
     "ngx-pagination": "^3.2.0",
     "ngx-smart-modal": "^7.4.0",
     "rxjs": "^6.3.2",
     "rxjs-compat": "^6.3.2",
-    "tinymce": "^5.0.4",
-    "tslint": "^5.11.0",
+    "tinymce": "^6.4.2",
+    "tslint": "^6.0.0",
     "web-animations-js": "^2.3.1",
-    "xlsx": "^0.17.0",
-    "zone.js": "^0.8.26"
+    "xlsx": "https://cdn.sheetjs.com/xlsx-0.19.3/xlsx-0.19.3.tgz",
```

Comment:
I think ideally we'd want to install from a git source if possible. Although it's probably fine - there's just slightly less visibility as to what's being installed.

Their repo is hosted with Gitea: https://git.sheetjs.com/sheetjs/sheetjs.git

Installing from any git-managed repo is the same. Instructions for Github would apply to Gitea too: https://stackoverflow.com/questions/17509669/how-to-install-an-npm-package-from-github-directly

Just an FYI